octree/include/pcl/octree/octree_pointcloud_occupancy.h

```diff
@@ -40,87 +40,97 @@
 
 #include <pcl/octree/octree_pointcloud.h>
 
-namespace pcl
+namespace pcl {
+namespace octree {
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/** \brief @b Octree pointcloud occupancy class
+ *  \note This pointcloud octree class generate an octrees from a point cloud
+ * (zero-copy). No information is stored at the lead nodes. It can be used of occupancy
+ * checks. \note The octree pointcloud is initialized with its voxel resolution. Its
+ * bounding box is automatically adjusted or can be predefined. \note \note typename:
+ * PointT: type of point used in pointcloud \ingroup octree \author Julius Kammerl
```

Comment:
Unfortunately `clang-format` doesn't do fantastic job at reflowing Doxygen comments. Here, for example, all `\note`s, `\ingroup`, and `\author` got jammed in a single line.

Reply:
Did you find any other instances? If no, I'll hunt them down since now I know what to look for.

Reply:
Oh, I did not go through the entire diff TBH. I expect many issues of this sort, just grep for Doxygen commands (`\xxx`) that are not at the beginning of the line.